docs/metadata_etl.rst

```diff
@@ -0,0 +1,196 @@
+all militate for a Metadata ETL approach. Rather than attempting to deploy static metadata
+stored in the repository, the product's automation should *extract* the Standard Value Set 
+metadata from the org, *transform* it to include the desired values (as well as all existing
+customization), and *load* the transformed metadata back into the org. CumulusCI now ships
+with a task, ``add_standard_value_set_entries``, that makes it easy to do just this:
+
+  .. code-block:: yaml
+
+    add_standard_value_set_entries:
+        options:
+            entries:
+                - fullName: "New_Value"
+                  label: "New Value"
+                  closed: False
+            api_names:
+                - CaseStatus
+
+This task would retrieve the existing ``Case.Status`` picklist value set from the org,
+add the ``New_Value`` entry to it, and redeploy the modified metadata - ensuring that
+the application's needs are met with a safe, minimal intervention in the target org.
+
+Standard Metadata ETL Tasks
+---------------------------
+
+CumulusCI includes several out-of-the-box Metadata ETL tasks:
+
+ - ``activate_flow`` activates Flows (or Processes).
+ - ``add_page_layout_related_lists`` adds Related Lists, with given fields, to Page Layouts.
+ - ``add_permission_set_perms`` adds FLS and Apex class access entries to Permission Sets.
+ - ``add_picklist_entries`` adds picklist entries to a custom picklist field.
+ - ``add_standard_value_set_entries`` adds entries to a Standard Value Set.
+ - ``assign_compact_layout`` assigns a Compact Layout to a given sObject.
+ - ``set_duplicate_rule_status`` activates and deactivates Duplicate Rules.
+ - ``set_field_help_text`` sets the Help Text of a given field (a non-push-upgradeable attribute).
+ - ``set_organization_wide_defaults`` sets the Org-Wide Defaults for one or more sObjects,
+   and waits until sharing recalculation completes before proceeding.
+
+Most Metadata ETL tasks accept the option ``api_names``, which specifies the scope of the operation.
+In most cases, more than one entity may be transformed in a single operation. Each task is a single
+atomic Metadata API deployment operation. Consult the Task Reference or use the ``cci task info``
+command for more information on the usage of each task.
+
+More Metadata ETL tasks are being added regularly, and the Metadata ETL framework
+makes it easy to add more tasks.
+
+Namespace Injection
+-------------------
+
+All out-of-the-box Metadata ETL tasks accept a Boolean ``managed`` option. If ``True``, CumulusCI
+will replace the token ``%%%NAMESPACE%%%`` in API names and in values used for transforming metadata
+with the project's namespace; if ``False``, the token will simply be removed.
+
+
+Implementation of Metadata ETL Tasks
+------------------------------------
+
+This section covers internals of the Metadata ETL framework, and is intended for
+users who wish to build their own Metadata ETL tasks.
```

Comment:
This section should probably eventually end up as a section of a larger topic on extending CumulusCI in Python and just be linked from here.